cyhy/mailer/CyhyMessage.py

```diff
@@ -61,6 +63,8 @@ class CyhyMessage(ReportMessage):
 
 <p>The Cyber Hygiene scan results are attached for your review. Same password as before. (If this is your first report and you have yet to receive a password, please let us know.)</p>
 
+<p><b>Additional details regarding findings, hosts, mitigated vulnerabilities, potentially risky services, etc., can be found under Appendix G: Attachments.</b> To access the attachments embedded within the report, open the report with a dedicated PDF reader (such as Adobe Acrobat), and click on the paper clip icon to the left of the attachment name.
```

Comment:
You should really close the `<p>` tag:
```suggestion
<p><b>Additional details regarding findings, hosts, mitigated vulnerabilities, potentially risky services, etc., can be found under Appendix G: Attachments.</b> To access the attachments embedded within the report, open the report with a dedicated PDF reader (such as Adobe Acrobat), and click on the paper clip icon to the left of the attachment name.</p>
```

You will need to update the test code accordingly.

Reply:
Also, please use more descriptive commit messages than "Update filename.py".  Describe what changes you're actually making.

Reply:
Done in commit 2af19ca.